docs/source/EccFramework.rst

```diff
@@ -0,0 +1,201 @@
+                aware simulation). Available: [AerSimulator('aer_simulator'), AerSimulator('aer_simulator_statevector'), AerSimulator('aer_simulator_density_matrix'), AerSimulator('aer_simulator_stabilizer'),
+                AerSimulator('aer_simulator_matrix_product_state'), AerSimulator('aer_simulator_extended_stabilizer'), AerSimulator('aer_simulator_unitary'), AerSimulator('aer_simulator_superop'),
+                QasmSimulator('qasm_simulator'), StatevectorSimulator('statevector_simulator'), UnitarySimulator('unitary_simulator'), PulseSimulator('pulse_simulator')]
+    -ecc ECC    Specify a ecc to be applied to the circuit. Currently available are Q3Shor, Q5Laflamme, Q7Steane, Q9Shor, Q9Surface, and Q18Surface (Default=none)
+    -fq FQ      Specify after how many qubit usages error correction is applied to it (Default=100)
+
+Available error-correcting codes
+#####################################################
+
+Properties
+----------
+
+.. list-table:: Properties of available error-correcting codes
+  :widths: 22 13 13 13 13 13 13
+  :header-rows: 1
+
+  * - Feature
+    - Q3Shor :cite:labelpar:`ShorCodes`
+    - Q5Laflamme :cite:labelpar:`LaflammeCodes`
+    - Q7Steane :cite:labelpar:`SteaneCode`
+    - Q9Shor :cite:labelpar:`ShorCodes`
+    - Q9Surface :cite:labelpar:`WoottonMinimalSurfaceCode`
+    - `Q18Surface <./EccFramework.rst#q18surface-code>`_
+  * - able to detect bit flips
+    - ✔️
+    - ✔️
+    - ✔️
+    - ✔️
+    - ✔️
+    - ✔️
+  * - able to detect phase flips
+    - ✖️
+    - ✔️
+    - ✔️
+    - ✔️
+    - ✔️
+    - ✖️*
+  * - #qubits for n logical qubits
+    - 3n+2
+    - 5n+4
+    - 7n+3
+    - 9n+8
+    - 9n+8
+    - 36n
+  * - #classical bits (total)
+    - 2
+    - 5
+    - 3
+    - 8
+    - 8
+    - 16
+
+\* Planned to work, but not fully implemented yet
+
+Available logical operations
+----------------------------
+
+.. list-table:: Available operations for each error-correcting code
+  :widths: 22 13 13 13 13 13 13
+  :header-rows: 1
+
+  * - Operation
+    - Q3Shor
+    - Q5Laflamme
+    - Q7Steane
+    - Q9Shor
+    - Q9Surface
+    - Q18Surface
+  * - Pauli (X, Y, Z)
+    - ✔️
+    - ✔️
+    - ✔️
+    - ✔️
+    - ✔️
+    - ✔️
+  * - controlled Pauli (CX,CY,CZ)
+    - ✔️
+    - ✖️
+    - ✔️
+    - ✔️
+    - ✔️
+    - ✖️
+  * - Hadamard
+    - ⚠️
+    - ✖️
+    - ✔️
+    - ✖️
+    - ✔️
+    - ✔️
+  * - S, S†, T, T†
+    - ✔️
+    - ✖️
+    - ✔️
+    - ✖️
+    - ✖️
+    - ✖️
+
+⚠️ = operation is applied without the scheme of the error-correcting code (i.e. decoding and encoding is performed before/afterwards, respectively, and the operation is encoded as-is)
+
+Q18Surface code
+---------------
+
+This code follows the principle of :cite:labelpar:`FowlerSurfaceCodes` and is applied to 18 data qubits.
```

Comment:
Seems a bit over the top to include this one-liner as an extra section down here. Just put the reference on top like the other references.

Reply:
Changed the lines as requested.